src/Mimi.jl

```diff
@@ -71,7 +71,7 @@ include("core/model.jl")
 include("explorer/explore.jl")
 include("mcs/mcs.jl")
 include("utils/graph.jl")
-include("utils/plotting.jl")
+# include("utils/plotting.jl")
```

Comment:
Just delete lines like that, there really is no need to keep stuff around in comments. We can always get old code back by looking at the git history.